exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -189,6 +189,15 @@ drill.exec: {
   debug: {
     return_error_for_failure_in_cancelled_fragments: false
   }
+  udf: {
+    retry.times: 5,
+    directory: {
+      base: "/tmp",
+      staging: ${drill.exec.zk.root}"/udf/staging",
```

Comment:
Using drill.dfs.home is not quite right, since I am not obliging user to use dfs. User can user fs, if he is running Drill in embedded more or just using one node (let's say for developing / testing purposes). Though I might use drill.fs.home and default it to /user/drill.

staging: ${drill.exec.zk.root}"/udf/staging" - corresponds to directory, not znode and I need ${drill.exec.zk.root} to separate jars from different Drill clusters.